.gitignore

```diff
@@ -2,6 +2,7 @@ node_modules/
 .yalc/
 yalc.lock
 dist
+build
```

Comment:
intentionally?

Reply:
@thekip Yeah. When I run e.g `yarn release:build` locally to test if builds work it creates a build folder 🤷 I can remove it if you'd like.

Reply:
Where exactly this folder is created?